e2e/cypress/integration/enterprise/extend_session/not_extended_when_disabled/with_saml_login_spec.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Requires openldap and keycloak running
+// Requires keycloak certificate at fixtures folder
+// - copy ./mattermost-server/build/docker/keycloak/keycloak.crt to ./mattermost-webapp/e2e/cypress/fixtures/keycloak.crt
+// Requires Cypress' chromeWebSecurity to be false
```

Comment:
@josephbaylon You'll need to manually copy the [keycloak.crt from server repo](https://github.com/mattermost/mattermost-server/blob/master/build/docker/keycloak/keycloak.crt) into the fixtures folder. Let me know if we need to modify the instructions above.

Reply:
My bad. I didn't notice the instructions.